test/jellyfish/config_reader_test.exs

```diff
@@ -71,6 +71,29 @@ defmodule Jellyfish.ConfigReaderTest do
     end
   end
 
+  test "read_check_origin/1" do
+    env_name = "JF_CHECK_ORIGIN"
+
+    with_env env_name do
+      System.put_env(env_name, "false")
+      assert ConfigReader.read_check_origin(env_name) == false
+      System.put_env(env_name, "true")
+      assert ConfigReader.read_check_origin(env_name) == true
+      System.put_env(env_name, "jellyfish.ovh jellyfish2.ovh jellyfish3.ovh")
+
+      assert ConfigReader.read_check_origin(env_name) == [
+               "jellyfish.ovh",
+               "jellyfish2.ovh",
+               "jellyfish3.ovh"
```

Comment:
Should we also allow for ip addresses?
Will we accept those values?

```elixir
check_origin: [
  "//another.com:888",
  "//*.other.com"
]
```

I took this from Phoenix docs https://hexdocs.pm/phoenix/Phoenix.Endpoint.html#socket/3-common-configuration

Reply:
I added test cases for this example from Phoenix docs. As for IP addresses, after another person's review, I removed any validation.